climada/util/config.py

```diff
@@ -321,3 +321,33 @@ def _fetch_conf(directories, config_name):
     Path(Path.home(), '.config'),  # ~/.config directory
     Path.cwd(),  # current working directory
 ], CONFIG_NAME))
+
+
+class log_level:
+    """Context manager that sets ALL loggers to a given level."""
+
+    def __init__(self, level, climada_only=True):
```

Comment:
I added the possibility to change the logging level of only CLIMADA loggers or of all loggers.